### ISSUE TITLE
Configurable auto-link

### ISSUE THREAD

**Issue by vitoque-git** [timbru31/quickjira#661]
Jira itself, is smart enough that if you have a ticket ID (like **MYREP-1234**), it would create a link from MYREP-1234 to the actual ticket.
Obvioulsy if you are outside on the web, in GMAIL, CHAT, or just an external website to Atlassian world that is not possible.
However it would be grand if this extension would do this.
I would think you need two pieces of configuration
1) A list of URL pattern in which this functionality is active
2) A list of project ID, ie ["MYPROJ","XYZ","COOLP"]

So if you are in one of the pattern specified in 1, and you find a string of the type COOLP-### then a link to the Jira via the issue is created when cliking  COOLP-###.
In other words
COOLP-###        ====>            <a href="LINK"> COOLP-###</a>

**Comment by timbru31** [comment]
Thanks for the suggestion, I'll explore the feasibility :)
